rules/style.js

```diff
@@ -10,8 +10,43 @@ module.exports = {
         'import/no-dynamic-require': 'off',
         'import/no-extraneous-dependencies': 'off',
         'import/no-unresolved': 'off',
-        // Enforce indentation of 4 spaces
-        indent: ['error', 4],
+
+        // Enforce indentation of 4 spaces; the third option parameter was copied from Airbnb:
+        // https://github.com/airbnb/javascript/blob/60b96d322277c4c71a21a05caba8eb3320e0e3fa/packages/eslint-config-airbnb-base/rules/style.js#L120-L145
+        indent: ['error', 4, {
+            SwitchCase: 1,
+            VariableDeclarator: 1,
+            outerIIFEBody: 1,
+            FunctionDeclaration: {
+                parameters: 1,
+                body: 1
+            },
+            FunctionExpression: {
+                parameters: 1,
+                body: 1
+            },
+            CallExpression: {
+                arguments: 1
+            },
+            ArrayExpression: 1,
+            ObjectExpression: 1,
+            ImportDeclaration: 1,
+            flatTernaryExpressions: false,
+
+            // list derived from https://github.com/benjamn/ast-types/blob/HEAD/def/jsx.js
```

Comment:
😅 lol airbnb has a comment in their code that doesn't work...

Reply:
😬 I didn't even check